src/widgets/aboutDialog.js

```diff
@@ -19,27 +19,11 @@
  */
 
 const {GObject, Gtk} = imports.gi;
+const ABOUT_RESOURCE = "/com/github/tchx84/Flatseal/widgets/aboutDialog.ui";
+function showAboutDialog(window) {
+    const builder = Gtk.Builder.new_from_resource(ABOUT_RESOURCE);
 
-
-var FlatsealAboutDialog = GObject.registerClass({
```

Comment:
Just curious, why not simply sub classing AdwAboutDialog here?

Reply:
AboutWindow and AboutDialog are final classes, they cannot be subclassed